src/server/lock_manager/config.rs

```diff
@@ -62,6 +65,7 @@ impl Default for Config {
             wake_up_delay_duration: ReadableDuration::millis(20),
             pipelined: true,
             in_memory: true,
+            wait_history_capacity: 0,
```

Comment:
Need a parameter or configuration to open it? Not open by default?

Reply:
IMO we should not open it by default until we are ready to GA the whole feature.